frontend/src/components/CategorySelect.tsx

```diff
@@ -2,16 +2,23 @@ import { useTranslation } from 'react-i18next';
 import { StringBank } from 'strings';
```

Comment:
Could you revert the changes of this file? they seem redundant. Or am I missing something here?
Also - Nadav is going to simplify this component

Reply:
done